source/gui/settingsDialogs.py

```diff
@@ -2050,6 +2050,13 @@ def __init__(self, parent):
 		self.UIAInMSWordCheckBox.SetValue(config.conf["UIA"]["useInMSWordWhenAvailable"])
 		self.UIAInMSWordCheckBox.defaultValue = self._getDefaultValue(["UIA", "useInMSWordWhenAvailable"])
 
+		# Translators: This is the label for a checkbox in the
+		#  Advanced settings panel.
+		label = _("Use UI Automation to access the Windows Console (highly experimental, restart NVDA for changes to take effect)")

```

Comment:
I think you can leave out the warning. The settings panel has quite a warning on it already. You can add a similar message to the user guide though.
```suggestion
		label = _("Use UI Automation to access the Windows Console.")
```